src/test/c_api_tests.cpp

```diff
@@ -41,14 +44,54 @@ static void testDefaultSingleModelOptions(MultiModelOptionsImpl* mmo) {
     EXPECT_EQ(mmo->idleSequenceCleanup, std::nullopt);
 }
 
+#define ASSERT_CAPI_STATUS_NULL(code)         \
+    {                                         \
+        auto* err = code;                     \
+        if (err != nullptr) {                 \
+            const char* msg = nullptr;        \
+            OVMS_StatusGetDetails(err, &msg); \
+            std::string smsg(msg);            \
+            OVMS_StatusDelete(err);           \
+            ASSERT_TRUE(false) << smsg;       \
```

Comment:
```suggestion
            ASSERT_EQ(err, nullptr) << smsg;       \
```

Reply:
You cannot assert for such equality, err is not nullptr. It was valid ptr prior line 54.

Reply:
done